lib/typesense/typesense-client.ts

```diff
@@ -14,4 +13,8 @@ export const typesenseSettings = {
 	],
 };
 
-export const client = new Client(typesenseSettings);
+export function createTypesenseClient(
+	apiKey: string = env.NEXT_PUBLIC_TYPESENSE_SEARCH_API_KEY,
+): Client {
```

Comment:
```suggestion
): TypesenseClient {
```

no need for the separate type import then